app/services/participant_validation_service.rb

```diff
@@ -55,7 +56,9 @@ def full_dqt_client
   end
 
   def matching_record(trn:, nino:, full_name:, dob:)
-    return if trn.blank?
+    return if trn.blank? && nino.blank?
```

Comment:
Are you sure we want to return here if the `nino` is blank? It contradicts a later line that handles it:

```ruby
nino_matches = nino.present? && nino.downcase == dqt_record["ni_number"]&.downcase
matches += 1 if nino_matches
```

Reply:
Sorry, @ethanmills pointed out this is an `&&` not `||`